Dockerfile

```
@@ -10,7 +10,8 @@ ENV NAMESERVERS="208.67.222.222 8.8.8.8 208.67.220.220 8.8.4.4" \
     PRE_RESOLVE=0 \
     MODE=tcp \
     VERBOSE=0 \
-    MAX_CONNECTIONS=100
+    MAX_CONNECTIONS=100 \
+    UDP_NO_ANSWERS=0
```

https://www.recallstack.icu/en/2020/09/01/do-not-use-negative-booleans-please/

```suggestion
    UDP_ANSWERS=1
```